packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -252,7 +317,7 @@ contract LockedGold is
    *   gold that has been unlocked but not yet withdrawn.
    * @return The total amount of locked gold in the system.
    */
-  function getTotalLockedGold() external view returns (uint256) {
+  function getTotalLockedGold() public view returns (uint256) {
     return totalNonvoting.add(getElection().getTotalVotes());
```

Comment:
I think it saves gas if you leave this as external and create a private method for accessing it.

Reply:
Actually, I don't see it used internally, so wondering if necessary..